### ISSUE TITLE
[Gradle Bug] Duplicate content roots detected

### ISSUE THREAD

**Issue by evowizz** [Deletescape-Media/Lawnchair#2647]
As of Android Studio Chipmunk (2021.2.1), source sets can no longer contain shared roots. However, Lawnchair still uses shared roots which causes Android Studio to not mark some folders as source sets, which itself causes syntax highlighting to be disabled.

You can find more details about this issue here:
ankidroid/Anki-Android#11291

And on Google's Issue Tracker:
https://issuetracker.google.com/issues/232007221#comment17

**Comment by suphon-t** [comment]
Syntax highlighting works fine for me on both Chipmunk and Dolphin. You should try clearing caches. 

Regardless, this is still an issue and should be fixed.

**Comment by evowizz** [comment]
Right, I talked with @patrykmichalik about it. Syntax highlighting also works on his end, but I've tried everything I could. Clearing Cache, resetting AS, etc. I suspect that it's actually because of some sort of cache that syntax highlighting still works for both of you, but I could be mistaken. 

All I can say for sure is that in my case most folders aren't marked as sources except for the ones under /tests, in which syntax highlighting works. And it seems to be related to the linked issues.
